permissions/plugin-apprenda.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "apprenda"
+paths:
+- "org/jenkins-ci/plugins/apprenda"
```

Comment:
Only correct if https://github.com/jenkinsci/apprenda-plugin/blob/e3781ea022adf8257f1caae3f512795967607877/pom.xml#L30 didn't exist.

Reply:
@daniel-beck should i remove line 30 from the BOM then?

Reply:
@michmike That, or adapt the YAML file in this PR. Either works.